.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-*                   @talkiq/engineering @TheKevJames
+*                   @talkiq/engineering @TheKevJames @cphoward
 
-pyproject.toml      @TheKevJames
-poetry.lock         @TheKevJames
+pyproject.toml      @TheKevJames @cphoward
+poetry.lock         @TheKevJames @cphoward
```

Comment:
Could you please remove these two lines? They're legacy, I had meant to remove 'em from all repos but must have missed a spot.